### ISSUE TITLE
Path effects applied to annotation text containing \n

### ISSUE THREAD

**Issue by dkirkby** [matplotlib/matplotlib#4024]
I am using the following test program with mathplotlib 1.4.0 (via anaconda) on OS X 10.10.1:
```
import matplotlib.pyplot as plt
import matplotlib.patheffects as pe

plt.subplot(1,1,1)
plt.annotate('line1\nline2',xy=(0.5,0.5),xycoords='axes fraction',fontsize='x-large',
	path_effects=[pe.withStroke(linewidth=1,foreground='r')])
plt.show()
```
This fails with:
```
AttributeError: GraphicsContextBase instance has no attribute 'draw_path'
```
Removing the \n from the text or removing the path_effects option both run as expected.  Any workarounds you can suggest are appreciated.

**Comment by cimarronm** [comment]
Definitely a problem with the MacOS backend and probably affects other backends which use a stack-based graphics context

**Comment by cimarronm** [comment]
@dkirkby This should be resolved with PR #4201 and #4202.

The workaround without these fixes applied is to separate out the annotation into separate annotations (without the newline).

**Comment by jenshnielsen** [comment]
I don't think this is fixed for OSX so I will reopen #4024 fixed it for Cairo

**Comment by tacaswell** [comment]
not blocking 1.5 on this unless @mdehoon wants to.
